clang/unittests/Format/FormatTest.cpp

```diff
@@ -26417,6 +26417,20 @@ TEST_F(FormatTest, BreakAfterAttributes) {
                "void g() {}",
                CtorDtorCode, Style);
 
+  Style.ReferenceAlignment = FormatStyle::ReferenceAlignmentStyle::RAS_Left;
```

Comment:
You are changing the style for all following tests. (Although it may not have an effect on them.) I don't think you should do that.

Put your test at the bottom, or restore the value.

Reply:
To restore the value, is it ok if I do it like this: `Style.ReferenceAlignment = getLLVMStyle().ReferenceAlignment;`?

Reply:
Changed verifyFormat -> verifyNoChange, to place the expected output only.
Restored value with `getLLVMStyle().ReferenceAlignment`, if this is not ok, I can substitute with `FormatStyle::ReferenceAlignmentStyle::RAS_Pointer`

Reply:
Fixed, Moved test to the bottom